### PR TITLE
feat: add echo project waitlist components

### PR DESCRIPTION
**Pull request by jainpawan21**
### What change does this PR introduce?
Add `EchoProjectWaitList` components

### CODE REVIEW

File: apps/web/src/pages/quick-start/components/ChannelsConfiguration.tsx [hunk 1]
```diff
@@ -32,6 +34,7 @@ export function ChannelsConfiguration({ setClickedChannel }: { setClickedChannel
       segment.track(OnBoardingAnalyticsEnum.ONBOARDING_EXPERIMENT_TEST_NOTIFICATION, {
         action: 'Get started - Send test notification',
         experiment_id: '2024-w9-onb',
+        _organizationId: currentOrganization?._id,
```
Comment (scopsy) [inline apps/web/src/pages/quick-start/components/ChannelsConfiguration.tsx:37]:
```suggestion
        _organization: currentOrganization?._id,
```

File: apps/web/src/pages/templates/components/TestWorkflow.tsx [hunk 2]
```diff
@@ -192,6 +192,7 @@ export function TestWorkflow({ trigger }) {
                   segment.track(OnBoardingAnalyticsEnum.ONBOARDING_EXPERIMENT_TEST_NOTIFICATION, {
                     action: 'Workflow - Run trigger',
                     experiment_id: '2024-w9-onb',
+                    _organizationId: currentOrganization?._id,
```
Comment (scopsy) [inline apps/web/src/pages/templates/components/TestWorkflow.tsx:195]:
```suggestion
                    _organization: currentOrganization?._id,
```

File: apps/web/src/pages/templates/workflow/WorkflowEditor.tsx [hunk 3]
```diff
@@ -283,6 +284,7 @@ const WorkflowEditor = () => {
                           segment.track(OnBoardingAnalyticsEnum.ONBOARDING_EXPERIMENT_TEST_NOTIFICATION, {
                             action: 'Workflow - Send test notification',
                             experiment_id: '2024-w9-onb',
+                            _organizationId: currentOrganization?._id,
```
Comment (scopsy) [inline apps/web/src/pages/templates/workflow/WorkflowEditor.tsx:287]:
```suggestion
                            _organization: currentOrganization?._id,
```

File: apps/web/src/pages/templates/components/EchoProjectWaitList.tsx [hunk 4]
```diff
@@ -0,0 +1,125 @@
...
+  return (
+    <TemplatesGroup key="new">
+      <GroupName>New</GroupName>
+      <ToolTip>
+        <TemplateItem
+          key="new-echo-project"
+          onClick={(event) => {
+            segment.track(SEGMENT_EVENT, {
+              action: segmentEventAction.dropDownItem,
+              _organizationId: currentOrganization?._id,
```
Comment (scopsy) [inline apps/web/src/pages/templates/components/EchoProjectWaitList.tsx:90]:
```suggestion
              _organization: currentOrganization?._id,
```

**Review by scopsy — APPROVED** [review]

File: apps/web/src/pages/templates/TemplatesListNoData.tsx [hunk 5]
```diff
@@ -146,6 +147,7 @@ export const TemplatesListNoData = ({
           <FontAwesomeIcon icon={faDiagramNext} />
           <span>All templates</span>
         </CardTile>
+        <EchoProjectCardTile />
```
Comment (LetItRock) [inline apps/web/src/pages/templates/TemplatesListNoData.tsx:150]: I would suggest moving this to the second position

Screenshot 2024-03-12 at 10 18 52

File: apps/web/src/pages/templates/components/EchoProjectWaitList.tsx [hunk 6]
```diff
@@ -0,0 +1,125 @@
...
+      Code Based
+    </>
+  );
+};
+
+const ToolTip = ({ children }) => {
+  return (
+    <Tooltip
+      label="Discover a new way that allows you to build powerful, highly customizable workflows on your local machine."
+      position="bottom"
```
Comment (LetItRock) [inline apps/web/src/pages/templates/components/EchoProjectWaitList.tsx:46]: i would suggest showing it on the right and please split the text to multiple lines

File: apps/web/src/pages/templates/components/templates-store/TemplatesStoreModal.tsx [hunk 7]
```diff
@@ -110,6 +111,23 @@ export const TemplatesStoreModal = ({ general, popular, isOpened, onClose }: ITe
...
+                  location: TemplateCreationSourceEnum.DROPDOWN,
+                });
+                handleRedirectToCreateBlankTemplate(false);
+              }}
+            >
+              <FontAwesomeIcon icon={faFile} />
+              <span>Blank Workflow</span>
+            </TemplateItem>
+          </TemplatesGroup>
+          <EchoProjectModalItem />
```
Comment (LetItRock) [inline apps/web/src/pages/templates/components/templates-store/TemplatesStoreModal.tsx:130]: I can suggest renaming the `Blank Workflow` group to the `Workflow` and adding the new Echo item under it, looks much better IMO.
Screenshot 2024-03-12 at 10 17 15

**Review by LetItRock — APPROVED** [review]

**Pull request merged**